.eslintrc

```diff
@@ -19,20 +19,11 @@
       "version": "detect"
     }
   },
-  "plugins": [
-    "react",
-    "cypress",
-    "react-hooks"
-  ],
-  "extends": [
-    "standard",
-    "standard-react"
-  ],
+  "plugins": ["react", "cypress", "react-hooks"],
```

Comment:
eh sorry, this came from me running Prettier's format on this file, nothing actually changing here

Reply:
did Prettier run on this file automatically (e.g. during the commit) because it was edited?

Reply:
I don't think so, I have prettier set up on my editor(atom) and ran it on save. It's currently set up to run using the `.editorconfig` file.

Reply:
OK. I think we can leave the formatting as-is.